bindings/wasm/src/crypto/wasm_signature_options.rs

```diff
@@ -37,6 +37,12 @@ impl From<SignatureOptions> for WasmSignatureOptions {
   }
 }
 
+impl From<&WasmSignatureOptions> for SignatureOptions {
+  fn from(options: &WasmSignatureOptions) -> Self {
+    options.0.clone()
+  }
+}
+
```

Comment:
No, I meant leave the original implementation of the owned type: `From<WasmSignatureOptions> for SignatureOptions`.